### PR TITLE
Bump version to v0.5.2

### PR DESCRIPTION
**Pull request by charliermarsh**

### CODE REVIEW

**Comment by charliermarsh** [comment]
Will release this in the morning.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,50 @@
...
+- Include nested configurations in settings reloading ([#12253](https://github.com/astral-sh/ruff/pull/12253))
+
+### CLI
+
+- Omit code frames for fixes with empty ranges ([#12304](https://github.com/astral-sh/ruff/pull/12304))
+- Warn about formatter incompatibility for `D203` ([#12238](https://github.com/astral-sh/ruff/pull/12238))
+
+### Bug fixes
+
+- Make cache-write failures non-fatal ([#12302](https://github.com/astral-sh/ruff/pull/12302))
```
Comment (zanieb) [inline CHANGELOG.md:32]: Just on Windows right?

**Review by zanieb — APPROVED** [review]

**Review by MichaReiser — APPROVED** [review]
Thanks!
@dhruvmanila should we also do an lsp/vs code release?

File: CHANGELOG.md (hunk 1 above)
Comment (charliermarsh) [reply CHANGELOG.md:32]: https://github.com/astral-sh/ruff/pull/12319

**Comment by dhruvmanila** [comment]
No, I don't think we should bump the version of VS Code extension, specifically because of https://github.com/astral-sh/ruff-vscode/pull/518 which we need to tie with the "stabilized" server release (https://github.com/astral-sh/ruff/pull/12208).

**Pull request merged**